hathor/cli/run_node.py

```diff
@@ -357,10 +359,14 @@ def __init__(self, *, argv=None):
         self.prepare()
         self.register_signal_handlers()
         if self._args.sysctl:
-            self.init_sysctl(self._args.sysctl)
+            if self._args.sysctl_config_file:
```

Comment:
We don't need this if block. Using `self.init_sysctl(self._args.sysctl, self._args.sysctl_config_file)` would work fine since `self._args.sysctl_config_file is None` when a file is not provided.